### ISSUE TITLE
GPT Embeddings + privacy?

### ISSUE THREAD

**Issue by SeaDude** [robertmartin8/MolecularNotes#2]
Hello,

Reading through the blog post (https://reasonabledeviations.com/2023/02/05/gpt-for-second-brain/) and curious how you’d handle private notes?

Maybe a yml tag in the front matter or…?

**Comment by robertmartin8** [comment]
Good point. I don’t use yaml because of weird interactions with backlinks, but I’d probably put a #private tag then filter these out
